pages/my-page/components/VoteNotification/index.tsx

```diff
@@ -0,0 +1,39 @@
+import Image from "next/image";
+
+import classNames from "classnames/bind";
+import dayjs from "dayjs";
+
+import { VoteNotificationType } from "@/types/type";
+
+import styles from "./VoteNotification.module.scss";
+
+dayjs.locale("ko");
+
+const cn = classNames.bind(styles);
+
+interface VoteNotificationProps {
+  notification: VoteNotificationType;
+}
+
+export default function VoteNotification({ notification }: VoteNotificationProps) {
+  const endDate = dayjs(notification.localDateTime).format("YY.MM.DD (ddd)");
+
+  return (
+    <div className={cn("container", { isVoted: notification.voteParticipateType })}>
```

Comment:
```suggestion
    <li className={cn("container", { isVoted: notification.voteParticipateType })}>
```
P5: 여기도 상위 태그에서 `ul`태그 사용하고 `li`태그를 사용하면 더 시맨틱해질 것 같아요! 추가적으로 여기서 클릭하면 투표 페이지로 이동하기 때문에 `li`태그 내부에 `Link`로 감싸서 페이지 이동까지 진행되어야 할 것 같아요~!